fp.js

```diff
@@ -31,4 +29,9 @@ var people = [
   {name: "Jessica", age: 34},
   {name: "Jasmine", age: 39}
 ]
-var peopleWhoseNamesHaveTheLetterS = people.filter(function(person){return person.name.match("s")});
+people.forEach(function(s){
```

Comment:
`.forEach` doesn't create a new array or modify the existing array here. The preferred approach is to `.push` onto our `peopleWhoseNamesHaveTheLetterS` array inside the if statement. Also the parameter to our callback function should be the current element of the `people` array:
```js
let peopleWhoseNamesHaveTheLetterS = []
people.forEach(function(person){
  if(person.name.match("s"))
    peopleWhoseNamesHaveTheLetterS.push(person)
})
```